### DISCUSSION TITLE
Revisions

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#15] [Assignments]
You can submit your revised memos and surveys in the thread below or email me directly.

**Comment by abi-200** [comment]
Memo (musicallamp): https://docs.google.com/document/d/1lBZaBSahF9hyGOtYPYIPV0T4lGln8qzQqhqSD0GYgcc/edit?usp=sharing

**Comment by edmondywen** [comment]
Little Tokyo Survey Revision: https://docs.google.com/forms/d/e/1FAIpQLSco7eZq2UwysCMNHfrKOtrfxFEdYA3WzZYmUJ23phUbrUz8aQ/viewform?usp=sf_link

**Comment by abi-200** [comment]
Survey (musicallamp):  https://forms.gle/Bb4R2ZMCBWZRb8cw9

**Comment by ronnie-blanche** [comment]
Survey (alphabetmafia): https://docs.google.com/forms/d/e/1FAIpQLSfhW_EGoXYwdjZUQ9YK_Ozvj8Qfrelxlu5ZQYGOyu1sYExCwg/viewform?usp=sf_link

**Comment by eandrusenki** [comment]
Survey (wehearseebelieveyou)
https://docs.google.com/forms/d/1M5VCeCdney14hRJc-e33pgDQ46nl5BabXLqdM87WXa4/edit?usp=sharing

**Comment by perezsara59** [comment]
Survey (income inequality):
https://forms.gle/Bsiwud66n8DE5Ju9A
